### DISCUSSION TITLE
v3.0.0-beta.1

### DISCUSSION THREAD

**Discussion by juliansteenbakker** [juliansteenbakker/mobile_scanner#310] [Announcements]
Breaking changes:
* [Android] SDK updated to SDK 33.

Features:
* [Web] Add binaryData for raw value.
* [iOS] Captures the last scanned barcode with Barcode.image.
* [iOS] Add support for multiple formats on iOS with BarcodeScannerOptions.
* Add displayValue which returns barcode value in a user-friendly format.
* Add autoResume option to MobileScannerController which automatically resumes the camera when the application is resumed

Other changes:
* [Android] Revert camera2 dependency to stable release
* [iOS] Update barcode scanning library to latest version
* Several minor code improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/juliansteenbakker/mobile_scanner/releases/tag/v3.0.0-beta.1'>v3.0.0-beta.1</a>.</em>

**Comment by jorgenpt** [comment]
Probably worth noting in the changelog that after #231 was merged there's now also permission callbacks! :)
